src/Common/TLDListsHolder.cpp

```diff
@@ -87,7 +87,7 @@ size_t TLDListsHolder::parseAndAddTldList(const std::string & name, const std::s
 
     size_t tld_list_size = tld_list.size();
     std::lock_guard<std::mutex> lock(tld_lists_map_mutex);
-    tld_lists_map.insert(std::make_pair(name, std::move(tld_list)));
+    tld_lists_map.insert(std::pair(name, std::move(tld_list)));
```

Comment:
We can use `emplace` and do argument perfect forwarding.

Reply:
`emplace` is surely able to replace `insert` (and `push_back`) in many cases. I would need to check if there is some automatic transform that does that (e.g. clang-tidy). To not blow PR further up, I would prefer to defer that to a separate PR.